### DISCUSSION TITLE
Emotional Dynamics in Generated Audio?

### DISCUSSION THREAD

**Discussion by shanecolb** [coqui-ai/TTS#1273] [General]
Hey everyone, 
I recently just got this working for temporary placement audio for a video game I am working on.  
It's a significant improvement from basic TTS systems, but I can't help wanting more....if possible.
The audio is relatively clear, and concise, but honestly, everything is so neutral sounding in dynamics. 
Is there a system in the works to allow emotional dynamics for text, to generate words or phrases with a specific emotional matrix? 
I understand this would be very difficult, given that the training data is all likely trained on neutral recordings, and not distinct recordings sets for Neutral, Scared, Sad, Happy etc. 
Cheers,
Shane

**Comment by erogol** [comment]
We do not have native support for emotional styles yet but are in the roadmap.
